src/cli.rs

```diff
@@ -149,8 +164,39 @@ pub enum IotHubDeviceUpdateConfig {
 }
 
 #[derive(Parser, Debug)]
-#[command(version, after_help = COPYRIGHT)]
-/// This tools helps to manage your omnect devices. For more information visit:\nhttps://github.com/omnect/omnect-cli
+#[command(after_help = COPYRIGHT)]
+/// establish ssh connections to devices
+pub struct SshConfig {
+    /// username for the login on the device.
+    #[arg(short = 'u', long = "user", default_value = "omnect")]
+    pub username: String,
+    /// optional: path where the ssh key pair, the certificates, and the
+    /// temporary ssh configuration is stored. Defaults to
+    /// ${XDG_RUNTIME_DIR}/omnect-cli, or /tmp/omnect-cli, if the xdg variable
```

Comment:
this will only work for linux systems or wsl2 then, right? do we know how this behaves on macos?

Reply:
dropped